app/shared/footer/_footer.scss

```diff
@@ -6,6 +6,13 @@ footer {
   text-align: center;
   font-size: 1.6rem;
 
+  .flex-container {
```

Comment:
Why do you need this flex-container? You are basically hacking the bootstrap column behavior.
If you want the columns to go on top of each other, then you should just make the columns 100% wide when in XS width. 
You can achieve this by adding `xs` prop to the columns e.g.: `<Col lg={3} md={3} xs={12}>` .

Reply:
It was some hack for fixing the alignment of the app version text to bottom. Now, I removed it and made some modification using media query and margin-top.